src/screens/game-details-modal/components/types/player.ts

```diff
@@ -0,0 +1,6 @@
+export type Player = {
+  picture: string;
```

Comment:
A player has not necessarily a picture. If he has no picture, show this.
<img width="125" alt="Capture d’écran 2024-01-18 à 12 56 39" src="https://github.com/wanned/karmine-corp-app/assets/57626613/204cdc06-d19d-4835-a799-17d82d5027f0">